AfsprakenRegels.md

```diff
@@ -781,6 +781,8 @@ Er kan sprake zijn van 1 naam en/of definitie die voor meerdere modelelementen g
 
 Echter, voor bijvoorbeeld voor een relatiesoort of een attribuutsoort kan het ook nodig zijn om dezelfde naam en/of definitie te specificeren. Dit is mogelijk, maar het is dan niet geheel duidelijk of er hetzelfde bedoeld wordt, of dat de (exacte) overeenkomt in een informatiemodel (zoals bv. in een UML informatiemodel) een toevalligheid is. Daarom gelden de volgende afspraken: 
 
+* Als de **uri** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het hetzelfde kenmerk is. Bijvoorbeeld bij objecttype Gebied met attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie" en een objecttype Boom met een attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie".
```

Comment:
Een URI is de identificatie van een modelelement (sectie 2.4.2.2). Dus twee modelelementen kunnen niet dezelfde URI hebben, want dan is zijn het dezelfde, dus maar 1 modelelement. Het gaat dan ook om meer dan kenmerken: dit geldt voor elk modelelement. Ik zou stellen: "Als bij twee modelelementbeschrijvingen dezelfde URI is opgenomen, dan wordt hiermee bedoeld dat het om hetzelfde modelelement gaat."

Reply:
In een modelelement zitten eigenlijk twee componenten. Dit komt duidelijk terug aan de LD kant; in [mim](https://docs.geostandaarden.nl/mim/mim/#structuur-metamodel-in-ld) wordt het vocabulaire en de shapesgraph (het schema) genoemd. Dit is logisch want we kunnen meerdere schema's (in LD de shapes) definieren voor één vocabulaire.

 Je kan er bijvoorbeeld voor kiezen om strenger te zijn in het waardebereik van een bepaalde eigenschap bij de ene objecttype dan bij een andere. Bijvoorbeeld voor nen3610:identificatie. Een model zou kunnen stellen dat de literal moet eindigen met "4". Elders wordt gestelt dat de literal moet eindigen met "2". De URI is in beide gevallen nen3610:identificatie. In deze twee modellen hebben we twee modelelement 'beschrijvingen' die conceptueel dezelfde eigenschap beschrijven, maar met een ander bijbehorend schema. Spreken we hier van hetzelfde modelelement of niet? We willen in iedergeval dat we in beide gevallen weten dat de literal een nen3610:identificatie van een object is. 

Ik weet dus niet of het handig is om te zeggen dat twee modelelementen waar dezelfde URI gebruikt wordt (in alle opzichten, vocabulaire EN schema) ook hetzelfde zijn, het is dus geen sameAs-achtige relatie.

Maar "kenmerk" lijkt inderdaad niet een optimale term, dat past misschien ook niet echt voor Objecttypes. Maar kunnen we wel modelelement gebruiken? Als dat vocabulaire én structuur omvat dan zijn twee modelelementen met dezelfde uri _niet_ noodzakelijkerwijs 'hetzelfde' (sameAs). 

Mijn voorstel is om in plaats van "kenmerk", "type" te noemen. Dan moet het wel algemeen duidelijk zijn/worden/kloppen dat met een mim model zowel een set aan "types" als (in meer of mindere mate) een bijbehorend schema specificeert.
Type is dan de verzamelterm voor wat we ookwel als rdfs:Class, rdf:Property of rdfs:Datatype zien.